### PR TITLE
[ODS-5727] Simplifications to API client experience using Profiles 

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: logistics/scripts/modules/postmanTestHarnessConfiguration.json [hunk 1]
```diff
@@ -11,14 +11,14 @@
           "ApiClients": [
             {
               "ApiClientName": "255901",
-              "LocalEducationOrganizations": [255901],
+              "LocalEducationOrganizations": [ 255901 ],
```
Comment (gmcelhanon) [inline logistics/scripts/modules/postmanTestHarnessConfiguration.json:14]: What tool did you use to format this file? It has introduced a lot of whitespace changes which create unnecessary noise in the history. All these whitespace changes should probably be reverted now (and not staged for the commit).

File: Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj [hunk 2]
```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="6.0.13" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.13" />
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="6.0.8" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.13" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.1" />
```
Comment (gmcelhanon) [inline Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj:29]: Why the downgrade here?

Comment (semalaiappan) [reply Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj:29]: 10>C:\OSS-Workspace\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.SandboxAdmin\EdFi.Ods.SandboxAdmin.csproj : warning NU1603: EdFi.Ods.Sandbox.Admin depends on Microsoft.Extensions.Caching.Memory (>= 6.0.13) but Microsoft.Extensions.Caching.Memory 6.0.13 was not found. An approximate best match of Microsoft.Extensions.Caching.Memory 7.0.0 was resolved.

File: logistics/scripts/modules/postmanTestHarnessConfiguration.json (hunk 1 above)
Comment (semalaiappan) [reply logistics/scripts/modules/postmanTestHarnessConfiguration.json]: I fixed whitespace issue in this file . Thanks

**Review by gmcelhanon — APPROVED** [review]

**Pull request merged**